packages/perseus/src/strings.ts

```diff
@@ -535,8 +535,8 @@ export const strings = {
         "The endpoint is at %(point1X)s comma %(point1Y)s and the ray goes through point %(point2X)s comma %(point2Y)s.",
     srRayGrabHandle:
         "Ray with endpoint %(point1X)s comma %(point1Y)s going through point %(point2X)s comma %(point2Y)s.",
-    srRayEndpoint: "Endpoint at %(point1X)s comma %(point1Y)s.",
```

Comment:
These use `x` and `y` in the `PerseusStrings` type and must match.

Reply:
Ouch! Sorry about that!